aedttest/aedt_test_runner.py

```diff
@@ -595,12 +596,12 @@ def extract_mesh_or_time_data(
                 # Check if profile exists already to avoid duplicates
                 cont = 1
                 filepath = new_absolute_path = Path(absolute_path)
-                while (self.reference_profiles / new_absolute_path.name).exists():
+                reference_profiles = Path(self.reference_folder, project_name)
```

Comment:
let's put it into subfolder in case we will want to change structure later, eg move .json into the same folder, or add new project files
```suggestion
                reference_profiles = Path(self.reference_folder, project_name, "prof")
```